_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,72 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Basic Setup
+---------------------------- */
+body {
+    font-size: $font-size;
+    min-width: $min-width;
+}
+
+body,
+input,
+select,
+textarea,
+button {
+    font-family: $font-sans;
+}
+
+a {
+    @include transition(color $timing-fastest ease-in-out 0s);
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+        text-decoration: underline;
+    }
+}
+
+h1,
+h2,
+h3 {
+    @include page-title('sans');
+}
+
+h4,
+h5,
+h6 {
+    @include page-title('serif');
+}
+
+p {
+    @include font-size();
+    @include line-height();
+}
+
+figure {
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+
+    figcaption {
+        @include line-height();
+        font-family: $font-serif;
+        font-style: italic;
+    }
+}
+
+.icon {
+    @extend %icon-font;
+}
+
+[class^="wrapper-"],
+[class*=" wrapper-"] {
+    @include clearfix();
+    @include box-sizing(border-box);
```

Comment:
Please remove, this is not needed as Bourbon has deprecated this utility (its natively supported enough now!) and our wildcard reset will take care of this.